### DISCUSSION TITLE
shen-ping/info/r6lfEKh7SVW83c6ErQvtD

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#6] [Announcements]
# shen-ping/info/r6lfEKh7SVW83c6ErQvtD

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/shen-ping/info/r6lfEKh7SVW83c6ErQvtD

**Comment by wurencaideli** [comment]
蒂花之秀
